aiogram/contrib/fsm_storage/memory.py

```diff
@@ -20,8 +20,8 @@ async def close(self):
     def __init__(self):
         self.data = {}
 
-    def resolve_address(self, chat, user):
-        chat_id, user_id = map(str, self.check_address(chat=chat, user=user))
+    def resolve_address(self, chat_id, user_id):
```

Comment:
Here and later: add 2 decorators
```python3
    @renamed_argument(old_name='user', new_name='user_id', until_version='3.0', stacklevel=3)
    @renamed_argument(old_name='chat', new_name='chat_id', until_version='3.0', stacklevel=4)
```
to all functions, in which you change the parameter names. This is required to keep the backward compatibility. We plan to completely remove this naming only in 3.0 (or not later than 2.12), so we need to allow both `user` and `user_id` for now

Reply:
Yep. Changing the naming of arguments in framework is breaking change and can not be done without deprecation process in minor releases.